scout/server/blueprints/variants/controllers.py

```diff
@@ -1007,27 +1007,32 @@ def check_form_gene_symbols(
 
     errors = {
         "non_clinical_symbols": {
-            "info": "Genes not included in gene panel versions in use when loading this case (clinical list).",
+            "message": "Genes not included in gene panel versions in use when loading this case (clinical list).",
             "gene_list": non_clinical_symbols,
+            "label": "info",
         },
         "not_found_symbols": {
-            "alert": "HGNC symbols not present in database genes collection",
+            "message": "HGNC symbols not present in database genes collection",
```

Comment:
HGNC symbols not present in the database's genes collection

Reply:
You are actually right. Now it's already merged unfortunately. And we are about to release. How about we fix it next PR that merge?